src/core/drive/preloader.ts

```diff
@@ -0,0 +1,54 @@
+import { Navigator } from "./navigator"
+import { PageSnapshot } from "./page_snapshot"
+import { SnapshotCache } from "./snapshot_cache"
+
+export interface PreloaderDelegate {
+  readonly navigator: Navigator
+}
+
+export class Preloader {
+  readonly delegate: PreloaderDelegate
+  readonly selector: string = 'a[data-turbo-preload="true"]'
```

Comment:
```suggestion
  readonly selector: string = 'a[data-turbo-preload]'
```

I think this is the preferred style here when a truthy attribute is expected, for example see https://github.com/hotwired/turbo/blob/8221c1e5f2021c41c3ecada0dde55bae9ee18380/src/core/snapshot.ts#L29